frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx

```diff
@@ -190,71 +128,48 @@ const InstallExternalCluster = withHandlePromise((props: InstallExternalClusterP
           </p>
+          <Alert
+            className="co-alert"
+            variant="info"
+            title="A bucket will be created to provide the OCS Service."
+            isInline
+          />
         </div>
         <Form className="im-install-page__form" onSubmit={onSubmit}>
-          <FormGroup
-            label="Cluster Name"
-            isRequired
-            fieldId="namespace-dropdown"
-            helperText={getErrorText(dataError.clusterName)}
-          >
-            <TextInput
-              onChange={(val) => setClusterName(val)}
-              value={clusterName}
-              aria-label="Enter Cluster name"
-              placeholder="openshift-storage"
-            />
-          </FormGroup>
-          <FormGroup
-            label="External FSID"
-            isRequired
-            fieldId="ext-fsid"
-            helperText={getErrorText(dataError.fsid)}
-          >
-            <TextInput
-              onChange={(val) => setExternalFSID(val)}
-              value={externalFSID ?? ''}
-              aria-label="Enter External FSID"
-              placeholder="asdf-ghjk-qwer-tyui"
-            />
-          </FormGroup>
-          <FormGroup
-            label="External admin secret"
-            isRequired
-            fieldId="ext-admin"
-            helperText={getErrorText(dataError.admin)}
-          >
-            <TextInput
-              onChange={(val) => setExternalAdminSecret(val)}
-              value={externalAdminSecret ?? ''}
-              aria-label="Enter Admin secret"
-              placeholder="!123jakajs==djjzla2"
-            />
+          <FormGroup label="External cluster metadata" isRequired fieldId="cluster-metadata">
+            <InputGroup>
+              <TextInput
+                aria-label="Upload JSON File"
+                value="Upload Credentials file"
+                className="im-install-page__input-box"
+              />
+              <FileUpload onUpload={onUpload} />
+            </InputGroup>
           </FormGroup>
-          <FormGroup
-            label="External mon data"
-            isRequired
-            fieldId="ext-mon-data"
-            helperText={getErrorText(dataError.monData)}
-          >
-            <TextInput
-              onChange={(val) => setExternalMonData(val)}
-              value={externalMonData ?? ''}
-              aria-label="Enter Mon Data"
-              placeholder="a='12.22.123.22'"
+          <FormGroup fieldId="preview-box">
+            <TextArea
+              value={prettifyJSON(fileData)}
+              className="im-install-page__text-box"
+              isValid={!dataError}
+              aria-label="JSON data"
             />
           </FormGroup>
-
-          <ButtonBar errorMessage={fileError || errorMessage} inProgress={inProgress}>
+          <ButtonBar errorMessage={dataError || errorMessage} inProgress={inProgress}>
             <ActionGroup>
               <Button type="submit" variant="primary">
```

Comment:
The button should be disabled if JSON is invalid, as user will be able be able to create cluster with invalid JSON.